app/mainwindow.cpp

```diff
@@ -500,8 +509,11 @@ void MainWindow::styleGithub()
 void MainWindow::styleSolarizedLight()
 {
     generator->setCodeHighlightingStyle("solarized_light");
-
-    ui->plainTextEdit->loadStyleFromStylesheet(":/theme/solarized-light+.txt");
+    if(options->isSourceAtOneSizeEnabled()) {
```

Comment:
I would prefer a separate private method to reduce this code duplication:

~~~cpp
void MainWindow::stylePath(const QString &styleName)
{
    QString suffix = options->isSourceAtOneSizeEnabled() ? "" : "+";
    return QString(":/theme/%1%2.txt").arg(styleName).arg(suffix);
}
~~~

Then this would become: `ui->plainTextEdit->loadStyleFromStylesheet(stylePath("solarized-light"));`